### PR TITLE
Add style for the uname class

### PR DESCRIPTION
**Pull request by xfq**

### CODE REVIEW

File: local.css [hunk 1]
```diff
@@ -144,6 +144,12 @@ a.termref:active {
 	border-bottom: 1px dotted #FC0; 
 	}
 
+.uname {
```
Comment (aphillips) [inline local.css:147]: match indentation of this file

Comment (xfq) [reply local.css:147]: I've tried to match the indentation of this file, but the indentation below seems wrong. I can modify the indentation of the code below if needed.

**Comment by xfq** [comment]
Merging per discussion with @r12a offline.

**Review by r12a — COMMENTED** [review]
LGTM

**Pull request merged**